Docs/sphinx_documentation/source/index.rst

```diff
@@ -55,6 +55,7 @@ Documentation on migration from BoxLib is available in the AMReX repository at D
    GPU_Chapter
    Visualization_Chapter
    Post_Processing_Chapter
+   Debugging
```

Comment:
Did you forget to add Debugging.rst to the commit?

Reply:
Yes! Thank you!